plugins/techdocs/src/home/components/Tables/EntityListDocsTable.tsx

```diff
@@ -65,6 +84,53 @@ export const EntityListDocsTable = (props: EntityListDocsTableProps) => {
     ),
   ];
 
+  const documents = entities.map(entity => {
```

Comment:
Hi, would it be possible for `CursorPaginatedDocsTable` and `OffsetPaginatedDocsTable` to use `DocsTable` so that the entity mapping logic is not duplicated? I noticed this same logic is also present in `DocsTable`  [here](https://github.com/backstage/backstage/blob/5550aa743348cec674e993d5d46298a99e6435b6/plugins/techdocs/src/home/components/Tables/DocsTable.tsx#L74).

Reply:
Sounds good!
I initially was going to remove the old <DocsTable /> and rename the <EntityListDocsTable /> to <DocsTable /> but decided against it for the breaking API changes.

Reply:
Hi! After looking at the code a little more closely, the `<DocsTable />` component seems rather limiting to me and a little awkward to be used to support pagination, and if it does then`<EntityListDocsTable />` provides little value.
I have a proposal - is it fair to deprecate the usage of the old `<DocsTable />` and promote the usage of the new `<EntityListDocsTable />` (as the new `<DocsTable />` perhaps).
If the main concern here is the duplication of the data mapping, perhaps extracting the logic in a little helper locally could help address that.
Let me know what you think!

Reply:
After looking at the signatures of both these components, it seems to make sense to leave both components as they are and allow folks to use whichever one they'd like to avoid breaking changes - this looks great!